app/models/user.rb

```diff
@@ -60,15 +59,16 @@ class User < ApplicationRecord
 
   attr_accessor :cgu_accepted
 
-  enum role: {
-    advisor: 'advisor',
-    admin: 'admin',
-    antenne_manager: 'antenne_manager'
-  }, _prefix: true
-
   ## Associations
   #
   belongs_to :antenne, inverse_of: :advisors
+  # rights / roles
```

Comment:
Sur les autres "sections" on a sauté une ligne avant le commentaire.
Si ça c'est pas du pinaillage 😅
```suggestion

  # rights / roles
```